src/loader-code.coffee

```diff
@@ -13,12 +13,14 @@ window.widgetLoader = ((window,document) ->
   defaults=
     widget_domain:  '//location.for.iframe.widget' # the contact form to load
     domain:         '//domain.for.iframe.widget'
```

Comment:
@yokosogithub we can probably leave just one of these above and specify here the api url where the data should be sent.